lib/libxx/Kconfig

```diff
@@ -61,6 +61,11 @@ if LIBCXX
 config LIBCXX_EXCEPTION
 	bool "Enable Exception Suppport"
 	default y
+	--help--
+		Enable exception support in libc++.
+
+		NOTE: C++ exception support requires toolchain version 10.3.1. If
+		you are	enabling C++ exception then use docker image version 2.0.0.
```

Comment:
use docker image version 2.0.0 by setting CONFIG_DOCKER_VERSION=2.0.0.

I suggest adding how to set at explanation.

Reply:
Done